cli/config/configfile/file.go

```diff
@@ -171,6 +171,10 @@ func (configFile *ConfigFile) Save() (retErr error) {
 	cfgFile := configFile.Filename
 	if f, err := os.Readlink(cfgFile); err == nil {
 		cfgFile = f
+		// The target of a symlink can be a relative path, ensure the final path is absolute
+		if !filepath.IsAbs(f) {
+			cfgFile = filepath.Join(dir, f)
+		}
 	}
```

Comment:
It's easier to just rely on the standard library to handle the symlinks:

```suggestion
	if f, err := filepath.EvalSymlinks(cfgFile); err == nil {
		cfgFile = f
	}
```

Reply:
I'm not very familiar with Go's standard libraries, but after reading the documentation and testing it locally, I found that `filepath.EvalSymlink` is a perfect solution to the problem. Thanks for the suggestion! I've updated the code.